scripts/generate/index.ts

```diff
@@ -7,26 +7,16 @@ import {extensionTypes} from './constants';
 import {addScripts} from './update-package-json';
 import {cleanUp} from './clean-up';
 
-const UNKNOWN_TYPE_ERROR = (type: string) => `
-Warning: Unknown extension point ${type}.
-Please use a supported extension type and generate your project manually.
-See README.md for instructions.
-`;
-
 interface InitConfig {
   type: string;
   template?: string;
 }
 
 async function init({type, template: templateIdentifier}: InitConfig) {
-  const extensionType = String(type);
-
-  if (extensionTypes.indexOf(extensionType) === -1) {
-    console.error(UNKNOWN_TYPE_ERROR(type));
-    process.exit(1);
-  }
+  const extensionType =
+    extensionTypes.find((t) => t === String(type)) || 'DEFAULT';
```

Comment:
In terms of UX, I would expect some more info if we're falling back to `DEFAULT`.

For example, if I try to create a `PRODUCT_SUBSCRIPTOIN` (with a typo) and I end up with a `DEFAULT` extension, I might be a bit confused 😕 .

Perhaps just `console.log` something like `Unknown extension type, ${type}, creating a ${extensionType} extension project instead`

Reply:
Great callout 👍 I'll add that in